ps/jenkins/package-testing-ps-8.0.groovy

```diff
@@ -70,6 +71,13 @@ pipeline {
             choices: ["yes", "no"],
             description: "check warning in client_test"
         )
+        
+        choice(
+            name: "install_mysql_shell",
+            choices: ["yes", "no"],
+            description: "install and check mysql shell for ps80 option"
```

Comment:
Is it only for ps80? It seems we have discussed that 1 option will be used for client_check and ps80?

Reply:
Yes this parameter will be used by both ps80 and client_check both.

Reply:
Then we should update option name and remove " for ps80 option"